src/test/java/uk/gov/ons/ctp/response/sample/endpoint/SampleEndpointUnitTest.java

```diff
@@ -80,8 +82,12 @@ public void uploadSampleFile() throws Exception {
         // Given
         String type = "B";
         MockMultipartFile file = new MockMultipartFile("file", "filename.txt".getBytes());
+        SampleSummary summary = new SampleSummary();
+        UUID summaryId = UUID.randomUUID();
+        summary.setId(summaryId);
```

Comment:
There's not much reason to assign the UUID to a variable here, it's not used further down.

Reply:
Done